modules/architecture-rhcos-updating-bootloader.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture_rhcos.adoc
+
+[id="architecture-rhcos-updating-bootloader.adoc_{context}"]
+= Updating the bootloader using bootupd
+
+To update the bootloader using bootupd, you must either install bootupd on {op-system} machines manually, or provide a machine config with the enabled systemd unit. Unlike grubby or other bootloader tools, bootupd does not manage kernel space configuration such as passing kernel arguments.
+
+Once installed, you can manage bootupd remotely from the {product-title} cluster.
+
+[NOTE]
+====
+It is recommended that bootupd is used only on bare metal or virtualized hypervisor installations to protect the boot process from a threat such as the BootHole vulnerability.
```

Comment:
Or maybe:
It is recommended that you use bootupd only on bare metal or virtualized hypervisor installations, such as for protection against the BootHole vulnerability.